frontend/bees-app/src/components/GameView.js

```diff
@@ -1,138 +1,85 @@
-import React from 'react';
+import React, { Component } from 'react';
 import '../App.css';
+import rapeseedFlower from '../rapeseed-flower.jpg';
+import acaciaFlower from '../acacia-flower.jpg';
+import lindenFlower from '../linden-flower.jpg';
+import falseIndigoFlower from '../false-indigo-flower.jpg'
+import sunFlower from '../sun-flower.jpg';
+import wildFlower from '../wild-flower.jpg';
+
+
+class CardGame extends Component {
```

Comment:
CardGame is not good name. Implies is a game of cards.

Reply:
change it in HiveCard